### ISSUE TITLE
gentoo installs fail due to invalid package atom

### ISSUE THREAD

**Issue by ITJamie** [saltstack/salt-bootstrap#1944]
### Setup + Steps to Reproduce Issue
using gentoo/stage3 container
attempt to bootstrap with `sh -XUdfPD -x python3 stable 3006.1`

```
#5 1168. + curl -L https://raw.githubusercontent.com/saltstack/salt-bootstrap/develop/bootstrap-salt.sh
#5 1168. + sh -s -- -XUdfPD -x python3 stable 3006.1
#5 1168.   % Total    % Received % Xferd  Average Speed   Time    T+ sed -e '/^\s\+echowarn "You have 10 seconds to cancel and stop the bootstrap process..."/,+2d'
#5 1168. ime     Time  Current
#5 1168.                                  Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0 *  INFO: Running version: 2023.06.28
#5 1168.  *  INFO: Executed by: 
#5 1168.  *  INFO: Command line: 'sh -XUdfPD -x python3 stable 3006.1'
#5 1168.  *  WARN: Running the unstable version of bootstrap-salt.sh
#5 1169. 
#5 1169.  *  INFO: System Information:
#5 1169.  *  INFO:   CPU:          GenuineIntel
#5 1169.  *  INFO:   CPU Arch:     x86_64
#5 1169.  *  INFO:   OS Name:      Linux
#5 1169.  *  INFO:   OS Version:   5.15.0-76-generic
#5 1169.  *  INFO:   Distribution: Gentoo 2.13
#5 1169. 
#5 1169.  * DEBUG: Binaries will be searched using the following $PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
#5 1169.  *  INFO: Installing minion
#5 1169.  *  INFO: Daemons will not be started
100  345k  100  345k    0     0   110k      0  0:00:03  0:00:03 --:--:--  110k
#5 1172.  * DEBUG: install_gentoo_2_onedir_deps not found....
#5 1172.  * DEBUG: install_gentoo_2_13_onedir_deps not found....
#5 1172.  * DEBUG: install_gentoo_2_deps not found....
#5 1172.  * DEBUG: install_gentoo_2_13_deps not found....
#5 1172.  * DEBUG: install_gentoo_onedir_deps not found....
#5 1172.  *  INFO: Found function install_gentoo_deps
#5 1172.  * DEBUG: DEPS_INSTALL_FUNC=install_gentoo_deps
#5 1172.  * DEBUG: config_gentoo_2_onedir_salt not found....
#5 1172.  * DEBUG: config_gentoo_2_13_onedir_salt not found....
#5 1172.  * DEBUG: config_gentoo_2_salt not found....
#5 1172.  * DEBUG: config_gentoo_2_13_salt not found....
#5 1172.  * DEBUG: config_gentoo_onedir_salt not found....
#5 1172.  * DEBUG: config_gentoo_salt not found....
#5 1172.  *  INFO: Found function config_salt
#5 1172.  * DEBUG: CONFIG_SALT_FUNC=config_salt
#5 1172.  * DEBUG: preseed_gentoo_2_onedir_master not found....
#5 1172.  * DEBUG: preseed_gentoo_2_13_onedir_master not found....
#5 1172.  * DEBUG: preseed_gentoo_2_master not found....
#5 1172.  * DEBUG: preseed_gentoo_2_13_master not found....
#5 1172.  * DEBUG: preseed_gentoo_onedir_master not found....
#5 1172.  * DEBUG: preseed_gentoo_master not found....
#5 1172.  *  INFO: Found function preseed_master
#5 1172.  * DEBUG: PRESEED_MASTER_FUNC=preseed_master
#5 1172.  * DEBUG: INSTALL_FUNC_NAMES=install_gentoo_2_onedir install_gentoo_2_13_onedir install_gentoo_onedir
#5 1172.  * DEBUG: install_gentoo_2_onedir not found....
#5 1172.  * DEBUG: install_gentoo_2_13_onedir not found....
#5 1172.  *  INFO: Found function install_gentoo_onedir
#5 1172.  * DEBUG: INSTALL_FUNC=install_gentoo_onedir
#5 1172.  * DEBUG: install_gentoo_2_onedir_post not found....
#5 1172.  * DEBUG: install_gentoo_2_13_onedir_post not found....
#5 1172.  * DEBUG: install_gentoo_2_post not found....
#5 1172.  * DEBUG: install_gentoo_2_13_post not found....
#5 1172.  *  INFO: Found function install_gentoo_onedir_post
#5 1172.  * DEBUG: POST_INSTALL_FUNC=install_gentoo_onedir_post
#5 1172.  * DEBUG: install_gentoo_2_onedir_restart_daemons not found....
#5 1172.  * DEBUG: install_gentoo_2_13_onedir_restart_daemons not found....
#5 1172.  * DEBUG: install_gentoo_2_restart_daemons not found....
#5 1172.  * DEBUG: install_gentoo_2_13_restart_daemons not found....
#5 1172.  * DEBUG: install_gentoo_onedir_restart_daemons not found....
#5 1172.  *  INFO: Found function install_gentoo_restart_daemons
#5 1172.  * DEBUG: STARTDAEMONS_INSTALL_FUNC=install_gentoo_restart_daemons
#5 1172.  * DEBUG: daemons_running_gentoo_2_onedir not found....
#5 1172.  * DEBUG: daemons_running_gentoo_2_13_onedir not found....
#5 1172.  * DEBUG: daemons_running_gentoo_2 not found....
#5 1172.  * DEBUG: daemons_running_gentoo_2_13 not found....
#5 1172.  * DEBUG: daemons_running_gentoo_onedir not found....
#5 1172.  * DEBUG: daemons_running_gentoo not found....
#5 1172.  *  INFO: Found function daemons_running_onedir
#5 1172.  * DEBUG: DAEMONS_RUNNING_FUNC=daemons_running_onedir
#5 1172.  * DEBUG: CHECK_SERVICES_FUNC=null
#5 1172.  *  INFO: Running install_gentoo_deps()
#5 1174. 
#5 1174. Nothing to sync... returning
#5 1174. 
#5 1174. 
#5 1174.  *  INFO: Running install_gentoo_onedir()
#5 1174. !!! '=app-admin/salt-minor/3006.1*' is not a valid package atom.
#5 1174. !!! Please check ebuild(5) for full details.
#5 1174.  * ERROR: Failed to run install_gentoo_onedir()!!!
#5 1174.  * DEBUG: Removing the logging pipe /tmp/bootstrap-salt.logpipe
#5 1174.  * DEBUG: Removing the temporary apt error file /tmp/apt_error.n2sqoo
```

3005.1: https://gitlab.com/ITJamie/salt-image-builder/-/jobs/4643607456#L347
3006.1: https://gitlab.com/ITJamie/salt-image-builder/-/jobs/4643607458#L389

**Comment by dmurphy18** [comment]
@ITJamie Can you recheck this with the just released v2024.12.12, lot of changes since then, but want to give you a heads up, discussion about keeping support for Gentoo, given it is a very minor usage for Linux (Broadcom since 2023)

**Comment by dmurphy18** [comment]
@ITJamie If you create a PR for what is missing, and fixes that, I will merge it. Don't have good way to test Gentoo (limited access to VMs and none now to AWS)
